schema/spacewalk/common/tables/suseContentProject.sql

```diff
@@ -0,0 +1,41 @@
+--
+-- Copyright (c) 2018 SUSE LLC
+--
+-- This software is licensed to you under the GNU General Public License,
+-- version 2 (GPLv2). There is NO WARRANTY for this software, express or
+-- implied, including the implied warranties of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+--
+-- Red Hat trademarks are not licensed under GPLv2. No permission is
+-- granted to use or replicate Red Hat trademarks that are incorporated
+-- in this software or its documentation.
+--
+
+CREATE TABLE suseContentProject(
+    id NUMBER NOT NULL
+        CONSTRAINT suse_ct_project_id_pk PRIMARY KEY,
+
+    name VARCHAR2(128) NOT NULL,
+
+    org_id NUMBER NOT NULL
+        CONSTRAINT suse_ct_project_oid_fk
+            REFERENCES web_customer(id)
+            ON DELETE CASCADE,
+
+    label VARCHAR2(16) NOT NULL,
```

Comment:
I think 16 is a bit too small.
I scanned some customer values and found some length a bit higher. I think with 20  or 24 most customers would be fine.

Of cause there are exceptions with prefixes of 45 characters.

Reply:
I see... I chose quite low value to prevent long names explosion in the channel names (max length there is 128). Let's hope 24 is still bearable :)